### PR TITLE
Enable ipv6 in orchagent

### PR DESCRIPTION
**Pull request by qiluo-msft**
Fix a bug in subnet() in previous pull request (already merged).

### CODE REVIEW

File: orchagent/swssnet.h [hunk 1]
```diff
@@ -83,13 +83,11 @@ inline static sai_ip_prefix_t& subnet(sai_ip_prefix_t& dst, const sai_ip_prefix_
     {
         case SAI_IP_ADDR_FAMILY_IPV4:
             dst.addr.ip4 = src.addr.ip4 & src.mask.ip4;
-            dst.mask.ip4 = 0xFFFFFFFF;
             break;
         case SAI_IP_ADDR_FAMILY_IPV6:
             for (size_t i = 0; i < 16; i++)
             {
                 dst.addr.ip6[i] = src.addr.ip6[i] & src.mask.ip6[i];
-                dst.mask.ip6[i] = 0xFF;
```
Comment (lguohan) [inline orchagent/swssnet.h]: what's the dst mask then?

**Comment by lguohan** [comment]
:clock1:

**Comment by lguohan** [comment]
can you first bring up your branch up-to-date with master and then generate the patch? there are too many commits in this pull request

File: orchagent/swssnet.h (hunk 1 above)
Comment (qiluo-msft) [inline orchagent/swssnet.h]: Resolved. Sorry for the mistake.

**Comment by qiluo-msft** [comment]
I already merged master to this pull request branch, so it has no conflict with master branch. I keep the old commits in case the original pull request is reverted. If you check 'Files changed', it will show the diff between master and the HEAD of this branch.

**Comment by lguohan** [comment]
Ack

**Comment by lguohan** [comment]
if you look at the commits, there are 11 commits.

**Pull request merged**